### ISSUE TITLE
Performance issues with many cats

### ISSUE THREAD

**Issue by TheShark27** [chris-blay/AndroidNougatEasterEgg#3]
Hello,
I have collected 97 cats, and now the scrolling is really laggy. The app has become unresponsive, just like the default Easter Egg has done.
Here's an example: https://youtu.be/qwjeTbgZKKE

Wouldn't it be better to add different pages of cats, or empty circles with the cats' color where the cat appears once the drawable is ready?
Thank you

**Comment by chris-blay** [comment]
Duplicate of #2

**Comment by TheShark27** [comment]
Ok, but may I know if it's possible to fix it?
